src/WebJobs.Extensions/Extensions/Timers/Scheduling/DailySchedule.cs

```diff
@@ -61,13 +61,13 @@ public override DateTime GetNextOccurrence(DateTime now)
                 // no more occurrences for today, so start back at the beginning of the
                 // the schedule tomorrow
                 TimeSpan nextTime = schedule[0];
-                DateTime nextOccurrence = new DateTime(now.Year, now.Month, now.Day, nextTime.Hours, nextTime.Minutes, nextTime.Seconds);
+                DateTime nextOccurrence = new DateTime(now.Year, now.Month, now.Day, nextTime.Hours, nextTime.Minutes, nextTime.Seconds, now.Kind);
```

Comment:
Now that I'm writing out the timezone offset, it exposed a little bug in how we created these new DateTimes. They'd end up with a Kind of `Unspecified`, which meant no timezone offset would appear when formatting the string. This changes it to copy the input value's `Kind` (which, for `DateTime.Now` is `Local`).